crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -149,21 +149,29 @@ pub fn update_frusta<T: Component + CameraProjection + Send + Sync + 'static>(
 
 pub fn check_visibility(
     mut view_query: Query<(&mut VisibleEntities, &Frustum, Option<&RenderLayers>), With<Camera>>,
-    mut visible_entity_query: Query<(
-        Entity,
-        &Visibility,
-        &mut ComputedVisibility,
-        Option<&RenderLayers>,
-        Option<&Aabb>,
-        Option<&NoFrustumCulling>,
-        Option<&GlobalTransform>,
+    mut visible_entity_query: ParamSet<(
+        Query<&mut ComputedVisibility>,
```

Comment:
Is there a reason we shouldn't just reuse the other query?

Reply:
mostly because this is what was done before #4489. Maybe to not iterate with all components?